plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestTimestampPrecision.java

```diff
@@ -91,15 +91,17 @@ public void testTimestampMicrosAsTimestampWithTimeZone(HiveTimestampPrecision ti
     }
 
     @DataProvider
-    public static Object[][] testTimestampMicrosDataProvider()
+    public static Object[][] testTimestampPrecisionDataProvider()
     {
-        return cartesianProduct(
-                new Object[][] {
-                        {HiveTimestampPrecision.MILLISECONDS, LocalDateTime.parse("2020-10-12T16:26:02.907")},
-                        {HiveTimestampPrecision.MICROSECONDS, LocalDateTime.parse("2020-10-12T16:26:02.906668")},
-                        {HiveTimestampPrecision.NANOSECONDS, LocalDateTime.parse("2020-10-12T16:26:02.906668")},
+        return cartesianProduct(new Object[][]{
+                        {HiveTimestampPrecision.MILLISECONDS, LocalDateTime.parse("2020-10-12T16:26:02.907"), "issue-5483.parquet"},
+                        {HiveTimestampPrecision.MICROSECONDS, LocalDateTime.parse("2020-10-12T16:26:02.906668"), "issue-5483.parquet"},
+                        {HiveTimestampPrecision.NANOSECONDS, LocalDateTime.parse("2020-10-12T16:26:02.906668"), "issue-5483.parquet"},
+                        {HiveTimestampPrecision.MILLISECONDS, LocalDateTime.parse("2020-10-12T16:26:02.907"), "issue-5483-nanos.parquet"},
```

Comment:
Why is the new file named `issue-5483-nanos.parquet` ? Not sure if it's strictly related to issue 5483
Maybe name it timestamp-nanos.parquet